encoding/kzg/verifier/multiframe.go

```diff
@@ -142,19 +96,14 @@ func genRhsG1(samples []Sample, randomsFr []fr.Element, m int, params encoding.E
 
 	// get leading coset powers
 	leadingDs := make([]fr.Element, n)
+	bigD := big.NewInt(int64(D))
 
 	for k := 0; k < n; k++ {
 
 		// got the leading coset field element
 		h := ks.ExpandedRootsOfUnity[samples[k].X]
 		var hPow fr.Element
-		hPow.SetOne()
-
-		// raising the power for each leading coset
-		for j := uint64(0); j < D; j++ {
-			hPow.Mul(&hPow, &h)
-		}
-
+		hPow.Exp(h, bigD)
 		leadingDs[k].Set(&hPow)
 	}
 
```

Comment:
nit: address the `UniversalVerify` comment at L158. And also the naming on L166 "group" -> "v"